src/containers/Genres/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import "./genres.scss"
+import { GenreItem } from "../../components/GenreItem";
+
+export const Genres:React.FunctionComponent = () => {
+    const genres = [
```

Comment:
Just another way to do the same:
```js
['All', 'Documentary', 'Comedy', 'Horror', 'Crime']
   .map((title, _id) => ({
      title,
      id: _id+1,
      active: !_id
   }))
```

Or, better, yet, if you would strong-type genres (using `enum Genre`), something like following would work:
```ts
Object
   .values(Genre)
   .map((title, _id) => ({
      title,
      id: _id+1,
      active: !_id
   }))
```